### ISSUE TITLE
Information on non-Minecraft libraries, shading, and JarJar

### ISSUE THREAD

**Issue by SizableShrimp** [MinecraftForge/Documentation#487]
Currently the "best" way to find out about the issues with non-Minecraft libraries on 1.17+ are to ask for help in the Forge discord and hope that someone who knows what the issue is runs `!nonmclibs` on you to explain the fix. This isn't optimal by any means; having to both join a discord & by chance have someone who knows what they are doing see your message.

`!nonmclibs` links to [this ForgeGradle wiki page](https://github.com/MinecraftForge/ForgeGradle/wiki/Dependencies#non-minecraft-dependencies)

We could improve this by also documenting it on the main docs. (I would assume the MinecraftForge doc subproject as this applies to modders and should have better visibility than the ForgeGradle doc subproject; where I don't know the intended purpose of that one)

Basically, everything on that ForgeGradle wiki page should probably have a home on the main docs somewhere that is more visible.

**Comment by Matyrobbrt** [comment]
Closed by https://github.com/MinecraftForge/Documentation/commit/e62a110c32a0c1b00389ad32c80f839576758550
